docs/reference/edot-sdks/nodejs/configuration.md

```diff
@@ -70,10 +70,11 @@ The behavior of `OTEL_` environment variables are typically defined by upstream
 | `OTEL_NODE_ENABLED_INSTRUMENTATIONS` 🔹 | [(EDOT Ref)](#otel_node_disabledenabled_instrumentations-details) Comma-separated list of instrumentations to enable. |
 | `OTEL_NODE_DISABLED_INSTRUMENTATIONS` 🔹 | [(EDOT Ref)](#otel_node_disabledenabled_instrumentations-details) Comma-separated list of instrumentations to disable. |
 | | |
-| `ELASTIC_OTEL_METRICS_DISABLED` 🔹 | [(EDOT Ref)](#elastic_otel_metrics_disabled-details) Disable any metrics being sent by the SDK. |
-| `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE` 🔹 | [(EDOT Ref)](#otel_exporter_otlp_metrics_temporality_preference-details) The metrics exporter's default aggregation `temporality`. Default `delta`. (Note: The upstream OTel default is `cumulative`.) |
+| `ELASTIC_OTEL_METRICS_DISABLED` 🔹 | [(EDOT Ref)](#deprecated-elastic_otel_metrics_disabled-details) Disable any metrics being sent by the SDK. This var is deprecated. |
```

Comment:
nit (non-blocking): (Doesn't have to be in this change.) I think it would be nice to make deprecated vars more obvious that they are deprecated. Perhaps putting them in a separate table and/or section below, or something.